packages/cordova/package.json

```
@@ -56,14 +56,14 @@
         "cordova-plugin-statusbar": "2.4.3",
         "cordova-plugin-x-socialsharing": "6.0.3",
         "es6-promise-plugin": "4.2.2",
-        "ionic-plugin-keyboard": "2.2.1",
         "typescript": "4.4.3"
     },
     "devDependencies": {
         "@types/cordova-plugin-qrscanner": "1.0.31",
         "clean-webpack-plugin": "3.0.0",
         "cordova": "11.0.0",
         "cordova-android": "10.1.2",
+        "cordova-plugin-keyboard": "^1.2.0",
```

Can you please fix this version? 
```suggestion
        "cordova-plugin-keyboard": "1.2.0",
```

Also, shouldn't this have been a normal dependency instead of a dev one?